tests/tests/operation_tests.cpp

```diff
@@ -4118,6 +4118,173 @@ BOOST_FIXTURE_TEST_SUITE(operation_tests, clean_database_fixture)
+            ACTORS((alice));
+            fund("alice", 1000000);
+
+            account_create_operation acc_create;
+
+            BOOST_TEST_MESSAGE("--- Creating account bob with alice");
+
+            const auto bob_private_key = generate_private_key("bob_active");
+            const auto bob_owner_authority = authority(1, generate_private_key("bob_owner").get_public_key(), 1);
+
+            acc_create.fee = ASSET("10.000 GOLOS");
+            acc_create.creator = "alice";
+            acc_create.new_account_name = "bob";
+            acc_create.owner = bob_owner_authority;
+            acc_create.active = authority(1, bob_private_key.get_public_key(), 1);
+            acc_create.posting = authority(1, generate_private_key("bob_posting").get_public_key(), 1);
+            acc_create.memo_key = generate_private_key("bob_memo").get_public_key();
+            acc_create.json_metadata = "";
+
+            signed_transaction tx;
+            BOOST_CHECK_NO_THROW(push_tx_with_ops(tx, alice_private_key, acc_create));
+
+            tx.clear();
+
+            BOOST_TEST_MESSAGE("--- Creating account chewy with alice");
+            acc_create.fee = ASSET("0.000 GOLOS");
+            acc_create.creator = "alice";
+            acc_create.new_account_name = "chewy";
+            acc_create.owner = authority(1, generate_private_key("chewy_owner").get_public_key(), 1);
+            acc_create.active = authority(1, generate_private_key("chewy_active").get_public_key(), 1);
+            acc_create.posting = authority(1, generate_private_key("chewy_posting").get_public_key(), 1);
+            acc_create.memo_key = generate_private_key("chewy_memo").get_public_key();
+            acc_create.json_metadata = "";
+
+            tx.operations.push_back(acc_create);
+
+            BOOST_TEST_MESSAGE("--- Creating account donna with alice");
+            acc_create.fee = ASSET("0.000 GOLOS");
+            acc_create.creator = "alice";
+            acc_create.new_account_name = "donna";
+            acc_create.owner = authority(1, generate_private_key("donna_owner").get_public_key(), 1);
+            acc_create.active = authority(1, generate_private_key("donna_active").get_public_key(), 1);
+            acc_create.posting = authority(1, generate_private_key("donna_posting").get_public_key(), 1);
+            acc_create.memo_key = generate_private_key("donna_memo").get_public_key();
+            acc_create.json_metadata = "";
+
+            tx.operations.push_back(acc_create);
+            tx.sign(alice_private_key, db->get_chain_id());
+
+            BOOST_CHECK_NO_THROW(db->push_transaction(tx, 0));
+
+            BOOST_TEST_MESSAGE("--- Set vesting withdraw routes");
+
+            set_withdraw_vesting_route_operation to_chewy;
+            to_chewy.from_account = "bob";
+            to_chewy.to_account = "chewy";
+            to_chewy.percent = 5000;
+
+            set_withdraw_vesting_route_operation to_donna;
```

Comment:
It is good to reuse one operation variable. 
```
set_withdraw_vesting_route_operation swvr_op;
swvr_op.from_account = "bob";
swvr_op.to_account = "chewy";
swvr_op.percent = 5000;
// and send it
// ...
swvr_opa.to_account = "donna";
// and push it
```

Reply:
and I want be able to use push_tx_with_ops()

Reply:
@e-schepachev No problems to use it.

Reply:
This cool method hides addition operations by hand 
            BOOST_CHECK_NO_THROW(push_tx_with_ops(tx, bob_private_key, to_chewy, to_donna));

less code and less code duplication.

Reply:
@e-schepachev No problems to use it with same operation. We have such codes.

Reply:
Oh, any call db->push_transaction() which is called in push_tx_whith_ops() creates new transaction in db. We can call push_tx_whith_ops() several times with same objects, but it will be different transactions in db. And operations will be in the different transactions.
Yes for this test there is no difference in one transaction will be operations or different. I want in one and what?

Reply:
@e-schepachev If few operations in one transaction are required - use different variables. (And it is good to fill transaction with `for` loop)
Otherwise use single operation variable.